Documentation/Signoffs/Head-On-Sensor.md

```diff
@@ -0,0 +1,59 @@
+# Head On Position Sensor
+## Functionality
+The purpose of the head on position sensor is to determine the target's location relative to the interceptor. Sensing equipment shall be attached to the front of the interceptor facing the test bed. The sensor will then read in data and make the necessary calculations to determine the X, Y, and Z position of the target. Higher accuracy is expected in the X and Y direction rather than Z, as to get an accurate reading on depth a time of flight sensor would be required with a high sampling rate, and this would be difficult and expensive given the test bed is only 6 feet long [1].
+## Constraints
+* C7: Design a sensor array that can detect approaching objects and relay their locations to the interceptor.
```

Comment:
This constraint is for the overarching system. Make this more applicable to the current signoff or omit the constraint alltogether

Reply:
That's a good point